Auth/Auth.Platform.Android/Auth.Platform.Android.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>AeroGear.Mobile.Auth.Platform.Android</id>
+    <version>0.0.1</version>
+    <title>AeroGear SDK Auth module (Android)</title>
+    <authors>AeroGear Community</authors>
```

Comment:
Actually there should be owners, I can add there some accounts we have on NuGet. Or we can register one AeroGear account to be overarching...